docs/develop/guides/working-with-momento-tokens.md

```diff
@@ -0,0 +1,131 @@
+2. Refresh Token: if you wish to programmatically obtain a new auth token before the current one expires, you need this refresh token.
+3. Valid Until: this is purely informational, and displays the date and time the current auth token expires at.
+
+![image](/img/getting-started/generated-token.jpg)
+
+## Step 3: Store your Momento Token
+If you wish to quickly test out Momento, click on the copy icon beside the auth token to copy the token to your clipboard and supply it to the Momento SDK. We recommend you also click the "Download JSON" button to store both the auth token and refresh token. Tokens are like passwords, the best practice is to store it in a secure location such as AWS Secrets Manager or GCP Secrets Manager.
+
+Here is an example of using the auth token in the Momento Javascript SDK, and then refreshing it to obtain a new token before it expires.
+<Tabs>
+    <TabItem value="nodejs" label="Node.js" default>
+
+```javascript
+const fs = require('fs');
+const { AuthClient, GenerateAuthToken, RefreshAuthToken, ExpiresAt, ExpiresIn } = require('@gomomento/sdk');
+const dotenv = require('dotenv');
+
+// Value must contain a valid go momento endpoint
+const controlEndpoint = "valid-gomomento-endpoint";
+// Value must contain a valid momento auth token
+const authToken = "valid-momento-auth-token";
+
+dotenv.config();
+
+// Creates the Momento auth client object
+async function createAuthClient() {
+  return new AuthClient();
+}
+
+// Creates the Momento cache client object
+async function createCacheClient(momentoAuthToken: string) {
+  return new CacheClient({
+    configuration: Configurations.Laptop.v1(),
+    credentialProvider: CredentialProvider.fromString({
+      environmentVariableName: momentoAuthToken,
+    }),
+    defaultTtlSeconds: 600,
+  });
+}
+
+async function refreshExpiringToken(authToken: string, refreshToken: string) {
+  const refreshTokenResponse = await authClient.refreshAuthToken(
+    CredentialProvider.fromString({
+      authToken: authToken,
+    }),
+    refreshToken
+  );
+
+  if (refreshTokenResponse instanceof RefreshAuthToken.Error) {
+    throw new Error(`Error: ${refreshTokenResponse.message()}`);
+  }
+  console.log('Auth token successfully refreshed!');
+  
+  return refreshTokenResponse;
+}
+
+async function usingAuthToken(cacheClient: CacheClient) {
+  const result = await cacheClient.listCaches();
+  if (result instanceof ListCaches.Success) {
+    console.log(
+      `Caches:\n${result
+        .getCaches()
+        .map(c => c.getName())
+        .join('\n')}\n\n`
+    );
+  } else if (result instanceof ListCaches.Error) {
+    throw new Error(
+      `An error occurred while attempting to list caches: ${result.errorCode()}: ${result.toString()}`
+    );
+  }
+}
+
+async function run() {
+  const byteArray = await readFile(filePath);
+  if (byteArray === null) {
+    return;
+  }
+
+  const authClient = await createAuthClient();
```

Comment:
@ben-kugler @cprice404 I don't understand what the purpose of `authClient` is in this example, do we need it to illustrate the purpose of auth token and refresh token?

Reply:
auth client is the class that provides the refresh/generate APIs.  See line 75.